service/controller/clusterapi/v29/resource/tccp/resource.go

```diff
@@ -33,29 +34,27 @@ const (
 // Config represents the configuration used to create a new cloudformation
 // resource.
 type Config struct {
-	APIWhitelist adapter.APIWhitelist
 	// EncrypterRoleManager manages role encryption. This can be supported by
 	// different implementations and thus is optional.
 	EncrypterRoleManager encrypter.RoleManager
 	Logger               micrologger.Logger
 
-	Detection                  *detection.Detection
-	EncrypterBackend           string
-	GuestPrivateSubnetMaskBits int
-	GuestPublicSubnetMaskBits  int
```

Comment:
That was so confusing. Since we have the ipam resource this configuration is of no use anymore.